### ISSUE TITLE
bubble are moving in slower rate in iPad, looks glitchy

### ISSUE THREAD

**Issue by pratibha691** [efremidze/Magnetic#78]
### New Issue Checklist

- [ ] I updated Magnetic to the latest version.
- [ ] I read the [Contribution Guidelines](https://github.com/efremidze/Magnetic/blob/master/.github/CONTRIBUTING.md).
- [ ] I read the [documentation](https://github.com/efremidze/Magnetic).
- [ ] I searched for [existing GitHub issues](https://github.com/efremidze/Magnetic/issues).

### Environment

- **iOS Version**: [INSERT iOS VERSION HERE]
- **Device(s)**: [INSERT DEVICE(S) HERE]

**Comment by vaibhavsingla1996** [comment]
I am facing the same issue. When I use this on iPad for first-time bubbles moves in a glitchy way.

**Comment by az-oolloow** [comment]
Yup, noticed the same on the simulator, the framerate was quite low.

**Comment by nipunV001** [comment]
I'm facing the same issue but I'm facing it in iPhone and iPhone simulator.
